src/portable/chipidea/ci_hs/ci_hs_imxrt.h

```diff
@@ -29,6 +29,11 @@
 
 #include "fsl_device_registers.h"
 
+#if CFG_TUSB_MCU == OPT_MCU_MIMXRT11XX
```

Comment:
for this we can replace it with 

```C
#if !defined(USB1_BASE) && defined(USB_OTG1_BASE)
#endif
```

NXP should really need to add USB1_BASE to USB_OTG1_BASE for backward compatible like stm does.

Reply:
Assuming that NXP will not do that, I'll declare this conversation as resolved.